File: graph2.py
v = int(input('Vetrex '))
e = int(input('Edges: '))
adj = [0]*v

for i in range(v):
    adj[i] = list()

# print(adj)
for i in range(e):
    a = int(input())
    b = int(input())
    adj[a].append(b)
    adj[b].append(a)
print('Adjancy list: ')
for i in range(v):
    print(i,":",end='')
    for j in adj[i]:
        print(j,"",end='')
    print()
    
    

File: graph2.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{
    int v,e;
    int a,b;
    cout<<"Enter total vertex and edges: ";
	cin>>v>>e;
	vector<int>adj[v];
	for(int i=0;i<e;i++){
		cout<<"Enter end points of edge"<<i+1<<": ";
		cin>>a>>b;
		adj[a].push_back(b);
		adj[b].push_back(a);
	}
	cout<<"Adj list:"<<endl;
	for(int i=0;i<v;i++){
		cout<<i<<": ";
		for(auto j:adj[i]){
			cout<<j<<" ";
		}
		cout<<endl;
	}

    
}